bitmovin/bitmovin-player-ios-analytics-conviva
# Support late player attaching

### Problem

There are certain scenarios where VST tracking should start before a Player instance is created. Currently there is no way to start a Conviva Session without a player instance.

### Solution

Extending the API to allow session initialization without a Player instance and adding a possibility to attach a Player instance at a later point in the life-cycle of the session.

### Notes

Manual session initialization API already existed and still needs to be used to start VST tracking.

### Manual testing instructions
I extended the Sample with a new `Start session` button. To test this, start the sample but first hit the `Destroy` button to stop and destroy the player/session, which automatically starts. (Of comment out `setupBitmovinPlayer` in `viewDidLoad`).
- Tap `Start Session` button
- Tap `Setup` button
- Double check in touchstone that the duration between those two actions is properly tracked as VST.

### Checklist
- [x] I added an update to `CHANGELOG.md` file
- [x] I ran all the tests in the project and they succeed
